src/relay/pass/fuse_ops.cc

```diff
@@ -731,6 +734,23 @@ GraphPartitioner::Partition(const IndexedForwardGraph& graph) {
   for (int phase = 0; phase < 2; ++phase) {
     this->RunFuse(graph, post_dom_tree, phase);
   }
+  // Fuse intermediate tuples, if any
+  for (size_t nid = 0; nid < graph.post_dfs_order.size(); ++nid) {
```

Comment:
can we fold this logic into RunFuse? (perhaps with different or same phase)

Reply:
done